aries-site/src/pages/components/button.mdx

```diff
@@ -56,6 +58,20 @@ Buttons are unique components since their alignment depends on where they appear
 
 <ButtonAlignmentTable />
 
+### Button Ordering
+
+The Primary button always comes first, followed by any Secondary buttons, then by any Default buttons.
```

Comment:
I like the succinctness of this sentence, however I am thinking of cases where a Primary button is not present, and speaking to the right-aligned scenario.

```suggestion
Button types may be applied in various combinations, however they should alway be ordered by their level of importance: Primary first, followed by Secondary, then by Default.

The order direction is determined by the button's alignment. Left-aligned buttons should be ordered from left to right; right-aligned, from right to left.
```

@CitizenFox, any suggestions?

Reply:
```suggestion
Button types may be applied in various combinations, however they should always be ordered by order of importance and in relation to their alignment: Primary first, followed by Secondary, then by Default.
```
I like your suggestion, @halocline. I added a few words to tie order to alignment. Does this make sense?